dev/TabView/TabView.cpp

```diff
@@ -270,6 +270,60 @@ void TabView::OnSelectedIndexPropertyChanged(const winrt::DependencyPropertyChan
     SetTabSeparatorOpacity(winrt::unbox_value<int>(args.OldValue()) - 1);
     SetTabSeparatorOpacity(SelectedIndex() - 1);
     SetTabSeparatorOpacity(SelectedIndex());
+
+    UpdateTabBorderVisualStates();
+}
+
+void TabView::UpdateTabBorderVisualStates()
+{
+    const int numItems = static_cast<int>(TabItems().Size());
+    const int selectedIndex = SelectedIndex();
+
+    for (int i = 0; i < numItems; i++)
+    {
+        auto state = L"NormalBorder";
+        if (m_isDragging)
+        {
+            state = L"NoBorder";
+        }
+        else if (selectedIndex != -1)
```

Comment:
I assume you tested the two scenarios where selectedIndex==0 and selectedIndex==numItems-1, right?

Reply:
Yes, certainly. :) If it's out of bounds ContainerFromIndex() returns null and then we don't do anything.

Reply:
I meant when selectedIndex==0, no tab will go into the LeftOfSelectedTab, and when selectedIndex==numItems-1, no tab will go into the RightOfSelectedTab state. I hope that's OK.

Reply:
Yes, that's all right -- the ItemsContainers handles the extra lines on the left and right in that case.